docs/using/administering/upgrading.rst

```diff
@@ -0,0 +1,34 @@
+=========
+Upgrading
+=========
+
+A Flocker cluster can be upgraded to a newer version of Flocker while preserving all data and configuration of the Flocker cluster provided certain steps are followed.
+The correct steps to follow may vary depending on the versions of Flocker being upgraded from and to.
+A common requirement for all currently supported upgrade paths is that all nodes and the control service must be running the same version of Flocker.
+
+Flocker 1.0.1
+-------------
+
+Recommended Steps
+^^^^^^^^^^^^^^^^^
+
+#. Stop the agent services on all nodes, and then stop control service.
+#. Install Flocker 1.0.1 on all nodes in the Flocker cluster.
+#. Restart the control service.
+#. If you are using the EBS storage backend reboot each of the agent nodes.
+#. If you have not configured the Flocker agents to start automatically on boot,
+   restart the agent services on all nodes.
+
+Details
+^^^^^^^
+
+The upgrade to Flocker 1.0.1 involves changing the way the EBS storage backend maps volumes to devices: in version 1.0.0, there were occasional errors in this mapping.
+As a result, some devices may be mounted in the wrong location.
```

Comment:
Small change:
As a result, some devices may have been mounted in the wrong location.